games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -30,10 +30,10 @@ index a67cd14..8493bb9 100644
  
  SRCS =	main.c		\
 -- 
-1.8.3.4
+2.7.0
 
 
-From 142ac2bc985591b85860f6fea82b5161fc4446e9 Mon Sep 17 00:00:00 2001
+From a90cc639c7268bb4ed9118aa691014ad61963161 Mon Sep 17 00:00:00 2001
```

Comment:
If you want to have the smallest git diff, you can replace "a90cc639c7268bb4ed9118aa691014ad61963161" by "142ac2bc985591b85860f6fea82b5161fc4446e9".